modules/setting/setting.go

```diff
@@ -248,6 +249,7 @@ var (
 		IssuePagingNum:      10,
 		RepoSearchPagingNum: 10,
 		MembersPagingNum:    20,
+		DirectoryPagingNum:  50,
```

Comment:
Default should be larger imho, I would say 100

Reply:
Have you tried this?

I thought 100 initially but actually 50 was long enough IMO. 

There are also a few performance arguments to be made for using a number <70.

Reply:
50 will make paging even for gitea repo first page...